jclklib/common/message.hpp

```diff
@@ -69,6 +69,7 @@ namespace JClkLibCommon
 		DECLARE_ACCESSOR(msgAck);
 		DECLARE_ACCESSOR(sessionId);
 
+		sessionId_t get_sessionId_val() const { return sessionId; }
```

Comment:
Hey Siang, i needed the const identifier for the function. As needed by the compiler . because it is being called by const object. hence we need the accessor to be const as well. 
So i did not want to change the DECLARE_ACCESSOR definition. 

Using the normal get_sessionId will have compilation failure.

Reply:
maybe i can try to use instead the getc_sessionID. however, i do think i did try but it didnt work on the compilation.

Reply:
![image](https://github.com/intel-staging/libptpmgmt_iaclocklib/assets/76154013/0bef2b18-a4c3-4bf2-be30-2bc925b192dd)

Reply:
i see. ok, we can focus on functional issue first, then code refactoring and clean up in later stage.

Reply:
keeping this error here as record tracking